sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala

```diff
@@ -262,61 +260,6 @@ class ThriftServerQueryTestSuite extends SQLQueryTestSuite with SharedThriftServ
     }
   }
 
-  /** Load built-in test tables. */
-  private def loadTestData(statement: Statement): Unit = {
-    // Prepare the data
-    statement.execute(
-      """
-        |CREATE OR REPLACE TEMPORARY VIEW testdata as
-        |SELECT id AS key, CAST(id AS string) AS value FROM range(1, 101)
-      """.stripMargin)
```

Comment:
Thanks for fixing it to loading the data only once!
But loading the data via JDBC statement was part of the testing coverage, to test executing those updates via JDBC. Instead of removing `loadTestData`, could you override `createTestTables` with creating and loading the data via JDBC statements?
While doing that, could you use `statement.executeUpdate` instead of `statement.execute`?

Reply:
hmm, I think we already have SQL tests for DDL/DML commands? If not we should add them.

Reply:
There are some tests that run a "CREATE TABLE" from within the test to set up some local data, but I believe not an extensive coverage.

Reply:
shall we create a test file to include all the temp views here and test reading/dropping?

Reply:
or we want to test `executeUpdate` explicitly?